README.md

```diff
@@ -90,33 +99,34 @@ Create a migration file. You can check the [examples](examples) folder for some
 
 ```json
 {
-  "name": "initial_migration",
-  "operations": [
-    {
-      "create_table": {
-        "name": "customers",
-        "columns": [
-          {
-            "name": "id",
-            "type": "integer",
-            "pk": true
-          },
-          {
-            "name": "name",
-            "type": "varchar(255)",
-            "unique": true
-          },
-          {
-            "name": "bio",
-            "type": "text",
-            "nullable": true
-          }
-        ]
-      }
-    }
-  ]
+	"name": "initial_migration",
+	"operations": [
+		{
+			"create_table": {
+				"name": "customers",
+				"columns": [
+					{
+						"name": "id",
+						"type": "integer",
+						"pk": true
+					},
+					{
+						"name": "name",
+						"type": "varchar(255)",
+						"unique": true
+					},
+					{
+						"name": "bio",
+						"type": "text",
+						"nullable": true
+					}
+				]
+			}
+		}
+	]
```

Comment:
this indentation change can be reverted.

Reply:
saved without formatting